### PR TITLE
lithuanian layout

### PR DESCRIPTION
**Pull request by MariusTrap**

### CODE REVIEW

**Comment by dessalines** [comment]
Check out some of the other language addition PRs. You need to add some things to Types.kt and Utils.kt

If you can't do this, lmk and I can.

**Review by dessalines — CHANGES_REQUESTED** [review]
See comment above.

**Comment by MariusTrap** [comment]
Changed Types.kt and Utils.kt. Didn't test anything.
Great concept and work.

File: app/src/main/java/com/dessalines/thumbkey/utils/Types.kt [hunk 1]
```diff
@@ -156,6 +156,7 @@ enum class KeyboardLayout(val title: String, val index: Int) {
     ThumbKeyHRv1Symbols("Thumb-Key hrvatski v1 with symbols", 58),
     TypeSplitFIv1("Type-Split suomi v1", 59),
     ThumbKeyLVLTGv1("Thumb-Key latviešu valoda v1", 60),
+    ThumbKeyLTv1("Thumb-Key lithuanian v1", 61),
```
Comment (dessalines) [inline app/src/main/java/com/dessalines/thumbkey/utils/Types.kt:159]: Use the non-english form: lietuvių kalba

File: app/src/main/java/com/dessalines/thumbkey/keyboards/ThumbKeyLTv1.kt [hunk 2]
```diff
@@ -0,0 +1,483 @@
...
+            BACKSPACE_KEY_ITEM,
+        ),
+        listOf(
+            SPACEBAR_KEY_ITEM,
+            RETURN_KEY_ITEM,
+        ),
+    ),
+)
+
+val THUMBKEY_EN_V4_SHIFTED = KeyboardC(
```
Comment (dessalines) [inline app/src/main/java/com/dessalines/thumbkey/keyboards/ThumbKeyLTv1.kt:246]: This variable name is wrong, make sure it compiles.

**Review by dessalines — APPROVED** [review]
Looks good, thx!

**Pull request merged**